index.js

```diff
@@ -66,8 +67,9 @@ class NodeClamError extends Error {
 // -----
 // @param   Object  options     Key => Value pairs to override default settings
 // ****************************************************************************
-class NodeClam {
+class NodeClam extends EventEmitter {
```

Comment:
`this.emit` is not available in the scope of `_process_result` so I made this class extend from `EventEmitter`.